README.md

```diff
@@ -99,16 +99,46 @@ const newBalance = await requestAndConfirmAirdropIfRequired(
 );
 ```
 
+### getExplorerLink(type, identifier, clusterName)
+
+Get an explorer link for an `address`, `block` or `transaction` (`tx` works too).
+
+```
+getExplorerLink(
+  "address",
+  "dDCQNnDmNbFVi8cQhKAgXhyhXeJ625tvwsunRyRc7c8",
+  "mainnet-beta",
+);
+```
+
+Will return `"https://explorer.solana.com/address/dDCQNnDmNbFVi8cQhKAgXhyhXeJ625tvwsunRyRc7c8"`. The cluster name isn't included since mainnet-beta is the default.
+
+```
+getExplorerLink(
+  "address",
+  "dDCQNnDmNbFVi8cQhKAgXhyhXeJ625tvwsunRyRc7c8",
+  "devnet",
+);
+```
+
+Will return `"https://explorer.solana.com/address/dDCQNnDmNbFVi8cQhKAgXhyhXeJ625tvwsunRyRc7c8?cluster=devnet"`
+
+```
+getExplorerLink("block", "241889720", "mainnet-beta");
+```
+
+Will return `"https://explorer.solana.com/block/241889720"`
+
 ## node.js specific helpers
 
-### getKeypairFromFile()
+### getKeypairFromFile(filename)
 
 Gets a keypair from a file - the format must be the same as [Solana CLI](https://docs.solana.com/wallet-guide/file-system-wallet) uses, ie, a JSON array of numbers:
 
 To load the default keypair `~/.config/solana/id.json`, just run:
 
 ```typescript
-const keyPair = await getKeypairFromFile();
+const keyPair = await getKeypairFromFile([file]);
```

Comment:
the `getKeypairFromFile` accepts a string string arg, not an array

Reply:
Good spot. I have no idea why I wrote that, guess that's why we have PRs.